rov/thrusters/Thrusters_Mock.py

```diff
@@ -0,0 +1,26 @@
+class Thrusters:
+
+    def __init__(self):
+        self.NUM_THRUSTERS = 8
+        self.thrusters = [0 for _ in range(0, self.NUM_THRUSTERS)]
+        #self._data["thrusters"] = { "actives": [], "force": [], "thrusters": {} }
+
+    @property
+    def data(self):
+        return self.thrusters
+
+    def get_data(self):
```

Comment:
We should remove this in favor of `@property def data(self):`, but the other thruster library had this so needed to be compliant..